app/views/users/_liquid.html.erb

```diff
@@ -15,7 +15,7 @@
     <h2><a href="<%= user.path %>" class="ltag__user__link"><%= user.name %></a><%= follow_button %></h2>
     <div class="ltag__user__summary">
       <a href="<%= user.path %>" class="ltag__user__link">
-        <%= user.summary %>
+        <%= user.summary_html.html_safe if user.summary_html.present? %>
```

Comment:
isn't the link going to be empty if there's no summary html?

Reply:
The reasoning behind this was if there is text in `user.summary` then there will text in `user.summary_html`.

Reply:
Not necessarily, summary_html is populate only before validation, which happens when the user is updated, which it won't happen for all existing users until they update their profile for something, hence `user.summary_html` will probably be empty for many for a while and this link will be empty. 

I think this should render summary if summary_html is empty

Reply:
That's true. I will make the change